crates/autopilot/src/infra/blockchain/contracts.rs

```diff
@@ -17,6 +18,7 @@ pub struct Contracts {
 pub struct Addresses {
     pub settlement: Option<H160>,
     pub weth: Option<H160>,
+    pub cow_amm_product_factory: Option<H160>,
```

Comment:
I think that cow amms operate with a constant product function is not very relevant here.

```suggestion
    pub cow_amm_factory: Option<H160>,
```